packages/components/pie-modal/src/index.ts

```diff
@@ -57,60 +132,59 @@ export class PieModal extends RtlMixin(LitElement) {
     }
 
     /**
-     * We require this because toggling the prop `isOpen` itself won't
-     * allow the dialog to open in the correct way (with the default background),
-     * the method `showModal()` needs to be invoked.
-     *
-     * https://lit.dev/docs/components/lifecycle/#willupdate
+     * Dismiss modal via `_handleCloseDialog()` on backdrop click
```

Comment:
I think this comment needs updating now?

Reply:
good spot